tests/all/unit/api/helpers/test_utilities.py

```diff
@@ -17,6 +17,20 @@ def test_get_filename_from_cd(self):
         self.assertEqual(expected_response_first, response_first)
         self.assertEqual(expected_response_none, response_none)
 
+    def test_string_empty(self):
+        """Method to test whether an empty string is correctly identified."""
+
+        self.assertTrue(string_empty(''))
+        self.assertTrue(string_empty(' '))
+        self.assertTrue(string_empty('  '))
+        self.assertTrue(string_empty('\n'))
+        self.assertFalse(string_empty(None))
```

Comment:
Should be true IMO

Reply:
`None` isn't str object I believe?

Reply:
A lot of times None can be passed for a string. We need to treat it as en empty string

Reply:
Travis was failing when I did that. What could be done?